nvfuser/__init__.py

```diff
@@ -189,10 +126,14 @@ def execute(
                 must either tell NVFuser where to run the resulting kernel, or
                 let it default to 0. Note that passing this option providing
                 and input tensors that lie on another device is an error.
+            print_repro (bool): Prints a reproduction script to stdout.
+            profile (bool): Captures a CUPTI based profile of a fusion.
```

Comment:
nitpick: order here isn't right. The two entry should be below `capture_debug_output`